java/src/jmri/jmrix/cmri/CMRISystemConnectionMemo.java

```diff
@@ -12,6 +13,8 @@
 
     public CMRISystemConnectionMemo() {
         super("C", CMRIConnectionTypeList.CMRI);
+        this.register();
```

Comment:
Curious why you put "this" in ```this.register();```.  I've seen people's conventions both with and without it.